bib_models/models/contacts.py

```diff
@@ -11,6 +11,8 @@ class Contact:
     """Contact information
     for a person or organization."""
 
+    street: List[str] = None
```

Comment:
Based on https://github.com/relaton/relaton-models/blob/main/models/Address.lutaml
and https://github.com/relaton/relaton-models/blob/main/grammars/biblio.rnc specs.


Same thing for attribute _postcode_